### DISCUSSION TITLE
ERROR: Disentangling failed: Failed in 'from scipy import stats, inf, log'!

### DISCUSSION THREAD

**Discussion by alikoraykoc** [Kinggerm/GetOrganelle#320] [Q&A] [answered]
Hello! 
I am trying to assemble example 1 but I am getting these errors:
ERROR: Disentangling failed: Failed in 'from scipy import stats, inf, log'!
INFO: Extracting embplant_pt from the assemblies failed.
I have reinstalled GetOrganelle twice. Also, tried the animal mitogenome example but I am getting the same errors. How can I fix this? Thank you.
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/14758803/get_org.log.txt)

**Comment by JianjunJin** [comment, marked as answer]
Hi @alikoraykoc 

Thanks for reaching out and sorry for the late reply. 
For now, please downgrade scipy from 1.12 to 1.11.3. See https://github.com/Kinggerm/GetOrganelle/issues/132#issuecomment-1958370216 for more.

**Reply by alikoraykoc** [reply]
Thank you for replying. I fixed the error by changing and adding some lines of code under the try statement which is in the "statistical_func.py" file. Changes that I made:
`try:`
`    from scipy import stats`
`    inf = float("inf")`
`    from math import log`
`except ImportError:`
Rest is the same with the original code.

**Reply by JianjunJin** [reply]
FYI, GetOrganelle 1.7.7.1, including compatibility with scipy 1.2, is now available in bioconda.
